common/jsonparser.cpp

```diff
@@ -24,7 +24,8 @@ namespace softwarecontainer {
 bool JSONParser::read(const json_t *element, const char *key, std::string &result)
 {
     json_t *value = json_object_get(element, key);
-    if (!value) {
+    if (value == nullptr) {
```

Comment:
If this is a better check, should the other similar checks below be changed too?